### ISSUE TITLE
How to train object detection model or classification model on custom dataset?

### ISSUE THREAD

**Issue by karndeepsingh** [Deci-AI/super-gradients#114]
Hi,
I have image data annotated for object detection as well on classification task. How I can train and build a object detection model and classification model on my own dataset?

A formal guideline would help me to understand the process thanks.  Basically, I want to know how we can load our custom data into the Dataloader in supergradient?

**Comment by shaydeci** [comment]
Hello,

If you already have a torch.dataset implementation for your data- you can pas them to DatasetInterface (see or documentation).

After that, try and follow the same procedure you see in our [walkthrough notebook example](https://colab.research.google.com/drive/1smwh4EAgE8PwnCtwsdU8a9D9Ezfh6FQK?usp=sharing) or our [Quickstart](https://colab.research.google.com/drive/12cURMPVQrvhgYle-wGmE2z8b_p90BdL0?usp=sharing).

Please let us know if you run into any problems.
